src/components/home/Service.js

```diff
@@ -1,7 +1,28 @@
+"use client";
 import React from "react";
-
+import { HiCodeBracket } from "react-icons/hi2";
 const Service = () => {
-  return <div>Service</div>;
+  return (
+    <div className="bg-winc-yellow w-96 h-96 rounded-lg  flex items-center justify-center">
```

Comment:
don't do w-96 or h -96 these are hard coded values. just do w-11/12(this is not a value but rather a proportion of how much space to take up) and h-full. Check the home/service component to see how the width and height work here.

Reply:
![image](https://github.com/acm-ucr/winc-website/assets/53918739/8f662f3b-ed68-4c28-9011-85c423ebeb84)
I made sure to change the w-11/12 but when I change the height the image is all weird, so I kept the 96 for the height.